sdk/monitor/monitor-query/src/generated/logquery/src/azureLogAnalyticsContext.ts

```diff
@@ -26,7 +26,7 @@ export class AzureLogAnalyticsContext extends coreClient.ServiceClient {
       requestContentType: "application/json; charset=utf-8"
     };
 
-    const packageDetails = `azsdk-js-monitor-log-query/1.0.0-beta.5`;
+    const packageDetails = `azsdk-js-monitor-log-query/1.0.0-beta.6`;
```

Comment:
@KarishmaGhiya Looks like each of the clients in this package is using a different user agent string. Please check with the feature crew if this is desired

Reply:
@KarishmaGhiya Any updates here?

Reply:
I had to separate out these 4 based on the different swaggers - otherwise was getting an error

Reply:
I will see how to use common user agent strings among these.

Reply:
Please log an issue for that investigation and continue with reviewing/merging this PR